python/paddle/trainer_config_helpers/layers.py

```diff
@@ -1566,13 +1566,21 @@ def expand_layer(input,
 
 @wrap_name_default()
 @layer_support()
-def repeat_layer(input, num_repeats, name=None, layer_attr=None):
+def repeat_layer(input,
+                 num_repeats,
+                 as_row_vector=True,
+                 name=None,
+                 layer_attr=None):
```

Comment:
The activation is implemented in C++ code. Should we add it to the Python interface?